core/src/com/unciv/ui/civilopedia/CivilopediaScreen.kt

```diff
@@ -283,6 +287,26 @@ class CivilopediaScreen(
                     )
                 }
 
+        if (!hideReligionItems)
+            categoryToEntries[CivilopediaCategories.Belief] = (
+                ruleset.beliefs.values.asSequence()
+                .map {
+                    CivilopediaEntry(
+                        it.name,
+                        "",
+                        CivilopediaCategories.Belief.getImage?.invoke(it.type.name, imageSize),
+                        (it as? ICivilopediaText).takeUnless { ct -> ct==null || ct.isCivilopediaTextEmpty() },
+                        sortBy = it.type.ordinal
```

Comment:
Nice!

Reply:
Some day we'll be able to do away with most of the dead or repeated code in there... Someday... In a future far far away...